src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -155,7 +164,12 @@ const Body = BaseForm.extend(Object.assign(
 const Footer = BaseFooter.extend({
   links () {
     let links = [];
-    const deviceChallenge = this.options.currentViewState.relatesTo.value;
+    this.deviceChallengePollRemediation = this.options.currentViewState;
+    const currentRemediationName = this.options.currentViewState.name;
+    let deviceChallenge = this.deviceChallengePollRemediation.relatesTo.value;
+    if (currentRemediationName === RemediationForms.USER_VERIFICATION_CHALLENGE_POLL) {
+      deviceChallenge = deviceChallenge.contextualData.challenge.value;
+    }
 
     if (deviceChallenge.challengeMethod === 'CUSTOM_URI') {
       links = [
```

Comment:
I guess in the UV case the link wont have to have href. Rather a form name. 'select-authenticator-enroll' I am assuming UV is always 

initial probe -> list of factors to verify (OV with UV is one of them) -> user selects OV (go back link shoudl take u to second step) 


If it makes sense you can use this 
https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/utils/LinksUtil.js#L57
or 
https://github.com/okta/okta-signin-widget/blob/0fc71d33b07b274019f606dd74ddd6f5470598a2/src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js#L73

Reply:
I will remove the go back link.

Reply:
@stevennguyen-okta can I have your input here?

Reply:
Yea we would need a Go Back button in some cases, for example these WIP SIW flows (ea) https://www.figma.com/file/VEFAgD40zwFS9kTJkrem3cg4/Okta-Verify-Library-WIP?node-id=5108%3A474 use a Go Back option for flows where you take substeps such as the alternate setup methods flows

Reply:
@stevennguyen-okta EA is out of scope for this PR, as well as device probing. For BETA, which user verification screen needs go back link?

Reply:
For Beta, the UV screens don't need a Go Back link, since they are in mid-auth using OV and we want to funnel them through if possible.  For reference during the UL flow when UV is needed: 
![image](https://user-images.githubusercontent.com/39841526/87593990-1f934180-c6a1-11ea-845d-9a7af0d513ca.png)

Reply:
@stevennguyen-okta the 'Extra verification needed' was what I implemented. But I thought after our last conversation, we decided to use the same mock for all the approaches for BETA. If this is not true, I can put back the code.

Reply:
@stevennguyen-okta  in ur mock posted above there could be cases where we have a intermediate screen where we list authenticators to authenticate with  (one of them being OV) in that case we need go back right. Note 2FA can be met with any authenticator right

![Screen Shot 2020-07-15 at 3 30 04 PM](https://user-images.githubusercontent.com/17267130/87607126-79036c80-c6b1-11ea-9d9d-b5b4a58c49da.png)